example/lambdas/db-provision-user-database/src/index.ts

```diff
@@ -48,7 +48,7 @@ export const handler = async (event: HandlerEvent): Promise<void> => {
       connection: config,
       acquireConnectionTimeout: 120000,
     });
-    const dbUser = event?.prefix.replace('-', '_');
+    const dbUser = event?.prefix.replace(/-/g, '_');
```

Comment:
This is the issue that's causing master to fail - stacks with multiple hyphens *obviously* won't the replacement done, and dashes are not allowed  in postgres for db names.